lib/Service/LockService.php

```diff
@@ -102,15 +103,22 @@ public function __construct(
 	 * @return void
 	 */
 	public function propFind(PropFind $propFind, INode $node) {
-		try {
-			$lock = $this->getLockFromCache($node->getId());
+		$propFind->handle(\OCA\FilesLock\AppInfo\Application::DAV_PROPERTY_LOCK, function () use ($node) {
+
+			try {
+				$lock = $this->getLockFromCache($node->getId());
 
-			if ($lock->getUserId() === $this->userId) {
-				return;
+				if ($lock->getUserId() === $this->userId) {
+					return false;
```

Comment:
@daita Is the lock only identified by the user id or is there a different token as well? 
Maybe we should just return the locking user by default, but if we also want additional details like the time when the lock happened whe need separate properties for that.

Reply:
I would say. Userid, displayname and lock time (or remaining) would make sense

Reply:
I will try to improve this, just that it was making the Text App not working as it tries to edit the content when you open the file.

Reply:
fixed!